askup/tasks.py

```diff
@@ -1,10 +1,17 @@
 from celery.schedules import crontab
 from celery.task import periodic_task
+from django.conf import settings
 
 from askup.utils.general import send_subscription_emails
 
 
-@periodic_task(run_every=crontab(hour='9', minute='0', day_of_week='4'))
+if 'SUBSCRIPTION_SCHEDULE' in dir(settings):
```

Comment:
Please use `getattr` method instead of all that big construction or better add `SUBSCRIPTION_SCHEDULE ` param with the default value to the base config file.